package.json

```diff
@@ -7,7 +7,7 @@
     "build": "npm run style:remove && npm run style:compile && npm run style:rename && npm run style:postCSS && npm run style:assets",
     "build:prod": "npm run build && npm run style:minify && npm run style:less",
     "deploy": "gh-pages -d docs/_site",
-    "docs:compile": "npm run docs:config && npm run build && npx gulp docs-build",
+    "docs:compile": "npm run build && npm run docs:config && cp scss/icons/SAP-icons.woff docs/css/; cp dist/fiori-fundamentals.css docs/css",
```

Comment:
I'm guessing the manual copying of these two files helps something along the way.  Is it just because of how they are imported and referenced within the core CSS file?

Reply:
Yeah I'm back and forth about doing it. I have them in the gitiginore, I was just trying to take some of the ~ mystery ~ out of what was happened/where everything was coming from. I believe we can reach into the dist folder and import` fiori-fundamentals.css` from there.

Reply:
It's not a big deal -- I was more just asking.  I think this looks pretty good.